covid19dp_submission/ingest_covid19dp_submission.py

```diff
@@ -51,24 +52,59 @@ def create_download_file_list(config: dict):
     return download_file_list
 
 
-def _get_config(snapshot_name: str, project_dir: str, nextflow_config_file: str, app_config_file: str) -> dict:
+def _write_properties(output_file, properties_str):
+    with open(output_file, 'w') as open_file:
+        open_file.write(properties_str)
+
+
+def _get_config(project: str, snapshot_name: str, project_dir: str, nextflow_config_file: str, app_config_file: str) -> dict:
     config = get_args_from_private_config_file(app_config_file)
 
     download_target_dir = os.path.join(project_dir, '30_eva_valid', snapshot_name)
+    release_dir = os.path.join(project_dir, 'release')
+    prop = SpringPropertiesGenerator(config['maven']['environment'], config['maven']['settings_file'])
+    accessioning_properties_file = os.path.join(download_target_dir, 'accessioning.properties')
+    clustering_properties_file = os.path.join(download_target_dir, 'clustering.properties')
+    release_properties_file = os.path.join(download_target_dir, 'release.properties')
+    _write_properties(accessioning_properties_file, prop.get_accessioning_properties(
+        target_assembly='GCA_009858895.3',
```

Comment:
Suggest making Covid assembly and taxonomy a top-level constant across the repo...

Reply:
Done